eng/common/scripts/Package-Properties.ps1

```diff
@@ -115,11 +121,17 @@ function Get-PkgProperties
     return $null
 }
 
-# Takes ServiceName and Repo Root Directory
 # Returns important properties for each package in the specified service, or entire repo if the serviceName is not specified
 # Returns an Table of service key to array values of PS Object with properties @ { pkgName, pkgVersion, pkgDirectoryPath, pkgReadMePath, pkgChangeLogPath }
-function Get-AllPkgProperties ([string]$ServiceDirectory = $null)
+# Pass the Package Name to filter down to one Package
+function Get-AllPkgProperties
 {
+    Param
+    (
+        [string]$ServiceDirectory = $null,
+        [string]$PackageName
+    )
```

Comment:
Should we also take ArtifactName here so that we can match on either?

Reply:
Especially given @praveenkuttappan's recent PR to add in artifact name https://github.com/Azure/azure-sdk-tools/pull/1374.